cddl/json.cddl

```diff
@@ -1,4 +1,5 @@
 ueid /= "ueid"
+nonce /= "nonce"
```

Comment:
Laurence, while you are on it:
* `debug-disble` should be `debug-disable`, and
* `nested-eat` should be deleted since it doesn't seem to be used

Reply:
Also, I think now we could uncomment `CDDL_FRAGS += json.cddl` in `cddl/vars.mk`